src/Settings.php

```diff
@@ -330,4 +338,14 @@ private function default() {
 			'custom_product_page_placement_priority' => 35,
 		);
 	}
+
+
+	/**
+	 * Get the enabled status for On-Site Messaging.
+	 *
+	 * @return bool
+	 */
+	public function is_enabled() {
+		return 'yes' === $this->settings['onsite_messaging_enabled'] ?? 'no';
```

Comment:
Change the logic here to default the value to `'yes'` if it does not exist, since we don't want to disable this for customers already using the feature.